build.sbt

```diff
@@ -22,12 +25,15 @@ lazy val root = project
     libraryDependencies += "commons-io" % "commons-io" % "2.16.1",
     libraryDependencies += "io.bullet" %% "borer-core" % "1.14.1",
     libraryDependencies += "io.bullet" %% "borer-derivation" % "1.14.1",
+    libraryDependencies += "com.palantir.isofilereader" % "isofilereader" % "0.6.1",
```

Comment:
Is this PR stacked on #31, or should we separate it from the partial ISO support?

Reply:
Stacked on #31, i forked from there…